EIPS/eip-2535.md

```diff
@@ -3,37 +3,37 @@ eip: 2535
 title: Diamonds, Multi-Facet Proxy
 author: Nick Mudge (@mudgen)
 discussions-to: https://github.com/ethereum/EIPs/issues/2535
-status: Last Call
-last-call-deadline: 2020-08-15
+status: Review
+description: A standard for creating modular smart contract systems that can be extended after deployment
 type: Standards Track
 category: ERC
 created: 2020-02-22
+requires: 1538, 2535
 ---
 
-<img align="right" src="../assets/eip-2535/diamond.svg" width="230" height="230">
-
-## Simple Summary
 
-Standard for creating modular smart contract systems that can be extended after deployment.
+## Abstract
+<img align="right" src="../assets/eip-2535/diamond.svg" width="230" height="230">
 
-Enables people to write smart contracts with virtually no size limit.
+This EIP proposes a standard for creating modular smart contract systems that can be extended after deployment ("Diamonds"). This enables smart contracts to effectively have no size limit, and to be upgraded without having to redeploy existing functionality. A Diamond's "Facets" can be added, replaced, or removed without having to worry about unintended side-effects, providing a layer of abstraction that aids development. 
```

Comment:
I appreciate your editing help and it is getting there.    This part is inaccurate: 
`A Diamond's "Facets" can be added, replaced, or removed without having to worry about unintended side-effects, providing a layer of abstraction that aids development.`

It is specifically functions which can be added/replaced/removed on diamonds,  not facets.  But functions that diamonds use exist in facets. For example it is possible to add some (but not all) functions from a facet to a diamond or remove only some functions from a diamond that exist in a facet which also has other functions which have not been removed from the diamond.  I don't want people to think (and some people have)  that all functions of a facet have to be added/replaced/removed in a diamond, because it is not true.

People do need to take care not to cause unintended side-effects from adding/replacing/removing functions in a diamond -- for example adding new functions with bugs that mess up the state in the diamond.

The challenges I have had when writing this specific EIP:
1. Help people really understand the standard technically and how it can be helpful.
2. Be both technically accurate and simple, but not so simple as to become technically inaccurate and misleading.